include/CLI/App.hpp

```diff
@@ -1148,9 +1164,41 @@ class App {
                 return subcomptr.get();
         throw OptionNotFound(subcom);
     }
+    /// Get a pointer to subcommand by index
+    App *get_subcommand(int index = 0) const {
```

Comment:
```suggestion
    App *get_subcommand(size_t index = 0) const {
```

This would remove the need for checking <0.

Reply:
I was trying to remember why I put it as an int.  Then I remembered that there is an overload that takes a pointer.  and if you pass a literal 0 you get a ambiguity, since 0 is also a valid pointer (gotta love C++) and if you gave it a NULL it would also be ambiguous. And that would just be annoying to use.  With int it matches the literal precisely and there is no ambiguity.  so you can call get_subcommand(0) and not get a compile error.

Reply:
Great, thanks C++. Okay.